core/templates/dev/head/css/oppia.css

```diff
@@ -1055,7 +1055,7 @@ textarea {
   background: linear-gradient(to right, rgba(0,0,0,0.65) 0%, rgba(0,0,0,0) 100%);
   /*The display is set in Library.js*/
   display: none;
-  height: 240px;
+  height: 259px;
```

Comment:
I did experiment with 258 first, but it didn't seem to have complete coverage (perhaps a fraction of a pixel).